grawlix-cms/_admin/book.view.php

```diff
@@ -497,7 +521,8 @@
 			$pagination[] = $link-> paint();
 		}
 	}
-	$pagination_output  = '<br/><p>Jump to '.implode(', ',$pagination).'</p>'."\n";
+	// Added easier page navigation links
```

Comment:
Another comment that's good to have during code review, but should be removed before merging, as it'll just be confusing to future maintainers.